.github/workflows/bazel_test.yml

```diff
@@ -23,11 +23,17 @@ jobs:
     steps:
       - name: Checkout repository
         uses: actions/checkout@v2
+      - name: Install deps 
+        run: |
+          sudo apt-get install python3-setuptools
+          pip3 install wheel absl-py
       - name: Run all tests
         working-directory: ./
         # Build //examples/...
         # Add "bazel test --config=clang //... --test_tag_filters=test"in the future if any test rules are added
         # The -FDP is for filtering out the files which contains "#include <fuzzer/FuzzedDataProvider.h>",
         # The "<fuzzer/FDP...>" is only supported since clang-10 while the clang version in github runner is 9
         run: |
+          alias python='python3'
           bazel build --verbose_failures --build_tag_filters=fuzz_test,-FDP --config=asan-libfuzzer //examples/...
+          bazel run //examples:empty_fuzz_test_run --config=asan-libfuzzer -- --timeout_secs=5
```

Comment:
It's great to test this command; I was wondering what we can do here to make the test fail if something other than a build goes wrong. Some failure scenarios in mind:

* The timeout is not obeyed, causing the test to run indefinitely. How will we catch this error? Can we make the command fail automatically (say if we leave it to run for 5 seconds, forcefully terminate it after 30 seconds) so it doesn't take too long to run? @asraa did Envoy have to come up with a solution to this problem?
* The command runs, but doesn't produce the expected output. For example, if we run a crashing fuzz target, running that command should expose the crash.

For the second problem, I imagine writing separate shell scripts and invoking them from here seems more scalable.

In any case, we can still push what we have now, and open a separate issue for improving it.

Reply:
We could add a timeout to the test script as a whole (like 5-10 min), which won't give fine-grained control over each test runtime, but would catch something like a fuzzer is running indefinitely 
(https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idtimeout-minutes)

For the second, I agree, having a framework to say "run this, expect with grep some error line comes up" would be cool